src/SonataBlockBundle.php

```diff
@@ -40,11 +40,13 @@ public function boot()
     /**
      * Register form mapping information.
      */
-    public function registerFormMapping()
+    private function registerFormMapping()
```

Comment:
Dont change the visibility , it is not needed

Reply:
Done.